collections/_posts/2022-06-28-tuto_git.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: review
+title: "tutu git"
+tags: CNN transformer classification
+author: "Olivier Bernard"
+cite:
+    authors: "Olivier Bernard"
+    title:   "Escaping the big data paradigm with compact transformers"
+    venue:   "CVPR LLID Workshop 2021"
+pdf: "https://arxiv.org/abs/2104.05704"
```

Comment:
Take care, bad link

Reply:
Hi sophie
Thanks for having poiting out this mistake. It is fixed now !